docs: DOC-121: Update Taxonomy tag page for external taxonomy

robot-ci-heartex opened the pull request:
Hi @caitlinwheeless!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/6398047806) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-frontend/pull/1577

caitlinwheeless committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6398047806

caitlinwheeless committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6398047806

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6398115755

caitlinwheeless committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6511594450

caitlinwheeless committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6511594450

caitlinwheeless commented:
/git merge develop

hlomzik commented:
/git merge develop
> **Error**: failed to execute "merge" command
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/6857400277)

hlomzik commented:
/frontend build lsf master
> **Error**: failed to get build
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/6857434199)

hlomzik committed:
Merge branch 'develop' into fb-docs/external-taxonomy

hlomzik commented:
/frontend build lsf master


> Successfully pushed new changes
>  heartexlabs/label-studio-frontend@10d60d6626b9a405cc6ba9ccbbe4b54d77e18278
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/6857489390)

hlomzik committed:
[frontend] Get build lsf

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6857489390

jombooth commented on docs/source/tags/taxonomy.md:
```diff
@@ -20,18 +22,19 @@ Use with the following data types: audio, image, HTML, paragraphs, text, time se
 | [placeholder=] | <code>string</code> |  | What to display as prompt on the input |
 | [perRegion] | <code>boolean</code> |  | Use this tag to classify specific regions instead of the whole object |
 | [perItem] | <code>boolean</code> |  | Use this tag to classify specific items inside the object instead of the whole object[^1] |
+| [legacy] | <code>boolean</code> |  | Use this tag to enable the legacy version of the Taxonomy tag. When true, the `apiUrl` parameter is not useable[^2] |
```
~please change "useable" to "usable" per GH Actions recommendation~ I've since learned that both spellings are acceptable so we can leave this
